index.html

```diff
@@ -17,23 +17,28 @@ <h1> What's for Dinner?</h1>
         <form id="form">
         <div>
         <p id="form-prompt"> <strong><p>What are you looking for?<span>*</span></strong></p>
-            <ul><input type="radio" id="side-option"
-            name="food" value="side-recipe">
+            <ul><input type="radio" id="sides-option"
```

Comment:
looks like you are missing an ending tag for `</main`. If you put that down on line 43 (right after the `</div>` tag), then you can use CSS to add flexbox to it and make your two main gray boxes (the What are you looking for? box and the box with the cookpot in it) go side-by-side like the comp shows.